### DISCUSSION TITLE
Cannot expand cross_sells.product

### DISCUSSION THREAD

**Discussion by siphosenkosindhlovu** [swellstores/community#27] [Help] [answered]
I've been trying to make a product page with support for cross sells, but I can't to expand the `products` property of cross_sells and keep getting the error shown below:

![image](https://user-images.githubusercontent.com/21548241/206399032-db087736-2d17-453b-97c1-fb3bbcd4b10d.png)

**Comment by alextutii** [comment, marked as answer]
Hi!

The Frontend API (swell-js) does not support expansion of `cross_sells.product`. Since you're making the request on the server, you could use the Backend API instead here as it supports this expansion. Here's a snippet of how that would look like with swell-node library:

```javascript
import swell from "swell-node";

export const getStaticProps = async ({ params }) => {
  swell.init(STORE_ID, STORE_SECRET_KEY);

  const product = await swell.get(`/products/${params.slug}`, {
    expand: ["cross_sells.product"],
  });

//...
})
```

**Reply by ericingram** [reply]
To clarify, it doesn't support expanding that field by default (but probably should) however it's possible to modify your API permissions to enable fields that aren't supported by default.

**Reply by y3owk1n** [reply]
> To clarify, it doesn't support expanding that field by default (but probably should) however it's possible to modify your API permissions to enable fields that aren't supported by default.

Hello @ericingram, just wondering, how can we modify the API permission? It would be great if you could point me to some docs. As I am looking to expand the product.reviews field as well.

**Reply by ericingram** [reply]
Here's a guide on that: https://developers.swell.is/guides/opening-content-permissions
